www/_posts/2022-09-09-media-plugin-6.1.0.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Media Plugin 6.1.0 Released!"
+categories: news
+tags: release plugins
+---
+
+We are happy to announce that we have just released an update for `cordova-plugin-media`!
+
+* [cordova-plugin-media@6.1.0](https://www.npmjs.com/package/cordova-plugin-media)
+
+**To upgrade:**
+
+```bash
+cordova plugin remove cordova-plugin-media
+cordova plugin add cordova-plugin-media@6.1.0
+```
+
+## Release Highlights
+
+For Android, we updated the media error response object to include the "message" property. This property will provide additional information about the error. It also syncs in line with the media error response object that iOS returns.
+
+For iOS, file scheme support has been implemented. Two fixes were applied to prevent crashes that occured when fetching the position while the time was not defined and to ensured that the category was defined after the recording was released.
+
+Please report any issues you find at [issues.cordova.io](http://issues.cordova.io/)!
```

Comment:
`issues.cordova.io` doesn't appear to exist, perhaps this should link to https://github.com/apache/cordova-plugin-media/issues instead?

Reply:
@breautek I updated the message and link.

IMO, I think it would be better to point to a generic link. Then the release blog post can be templated.

If the link was specific, anyone could accidentally forget to update the link when using the template.

Also, if the blog post was covering multiple package release, at the same time, then we have to change the entire message or repeat the sentence under each section.

Let me know what you think of the updated message, and what it points to.

Reply:
I think that's fine. My main concern was the broken link.